### PR TITLE
PR for the mergedSchemaTransforms proposal

### PR DESCRIPTION
**Pull request by rajeshdavidbabu**
This adds the changes and tests for the mergedSchemaTransforms proposal https://github.com/spring-media/themis-graphql/pull/72/files?short_path=9890705#diff-98907056bf54cfc1fea905a45251a6e2

And some lint fixes

### CODE REVIEW

File: src/strategies/all-local.js [hunk 1]
```diff
@@ -7,12 +7,10 @@ const {
 const { insertIfValue, insertIf } = require('../utils');
 
 module.exports = {
-  merge (srcs, { filterSubscriptions, logger }) {
+  merge (srcs, { filterSubscriptions, mergedSchemaTransforms = [], logger }) {
     const {
       typeDefs,
       resolvers,
-      extendTypes,
-      extendResolvers,
```
Comment (feitm) [inline src/strategies/all-local.js]: modules.md -->
Modules can be either local type definitions and resolvers, or a [remote schema](./remote-module). Both can be freely combined and stitched together.

A module can expose the following values:
- name
- typeDefs
- resolvers
- extendTypes
- extendResolvers

Was it planned to remove these?

Comment (rajeshdavidbabu) [reply src/strategies/all-local.js]: Oh this is just an ES Lint issue. These variables were not being used :)

File: cli.test.js [hunk 2]
```diff
@@ -194,6 +194,70 @@ describe('CLI', () => {
...
+      query: `query {
+        __schema {
+          types {
+            name
+          }
+        }
+      }`,
+    };
+
+    const res1 = await request('http://127.0.0.1:54328')
```
Comment (TimSusa) [inline cli.test.js]: strange naming

Comment (rajeshdavidbabu) [reply cli.test.js]: Every other test has the same naming. Let's keep it consistent and decide on changing it later

**Pull request merged**